kratos/processes/generic_find_elements_neighbours_process.h

```diff
@@ -0,0 +1,228 @@
+#if !defined(KRATOS_GENERIC_FIND_ELEMENTAL_NEIGHBOURS_PROCESS_H_INCLUDED )
+#define  KRATOS_GENERIC_FIND_ELEMENTAL_NEIGHBOURS_PROCESS_H_INCLUDED
+
+
+
+// System includes
+#include <string>
+#include <iostream>
+#include <unordered_map>
+
+// External includes
+
+
+// Project includes
+#include "includes/define.h"
+#include "processes/process.h"
+#include "processes/find_global_nodal_elemental_neighbours_process.h"
+#include "includes/node.h"
+#include "includes/element.h"
+#include "includes/model_part.h"
+#include "utilities/parallel_utilities.h"
+
+
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+typedef  ModelPart::NodesContainerType NodesContainerType;
+typedef  ModelPart::ElementsContainerType ElementsContainerType;
+typedef Geometry<Node < 3 > > GeometryType;
```

Comment:
These types must be defined inside the class. Note that you're defining them for the entire Kratos in here!

Reply:
ok, fixing it